medical_base_us/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Please update this to the new ReadMe template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst